.github/workflows/covector-version-or-publish.yml

```diff
@@ -127,6 +127,18 @@ jobs:
         with:
           name: artifacts-${{ matrix.config.arch }}
           path: "${{ join(fromJSON(steps.tauri_build.outputs.artifacts), '\n') }}"
+      - name: Rename elk binary to represent it's architecture and os
+        if: "matrix.config.os == 'ubuntu-latest'"
+        run: |
+          pwd
+          mkdir target/${{ matrix.config.rust_target }}/release/bundle/bin -p
+          cp ./target/${{ matrix.config.rust_target }}/release/elk ./target/${{ matrix.config.rust_target }}/release/bundle/bin/Elk_${{ needs.version-or-publish.outputs.version }}_${{ matrix.config.arch }}
```

Comment:
quick question, the naming scheme we have now is `Elk_${{version}}_${{os}}_${{arch}}` can we name these artifacts the same way or would that be incompatible with packaging? Bc I kinda feel like these names will cause confusion when being uploaded to GH releases

Reply:
I don't think that would be a problem. PKGBUILDs are basically bash scripts that move stuff to the correct folders. I'll update it later today

Reply:
nvm, actually had a few spare minutes :D